generate/resources/sync-gateway/README.md

```diff
@@ -170,7 +170,7 @@ Create a `/tmp/my-sg-config.json` file on your host machine, with the following:
 Start a Sync Gateway container in the `couchbase` network and use the `/tmp/my-sg-config.json` file:
 
 ```
-$ docker run --net=couchbase -p 4984:4984 -d -v /tmp:/tmp/config couchbase-sync-gateway /tmp/my-sg-config.json
+$ docker run --net=couchbase -p 4984:4984 -v /tmp:/tmp/config -d couchbase-sync-gateway /tmp/config/my-sg-config.json
```

Comment:
The issue isn't so much where it lives, and more that before it was inconsistent in the same line. We mapped `/tmp` on the host to `/tmp/config` on the container, but then looked for `/tmp/my-sg-config.json` when we'd put it in `/tmp/config/my-sg-config.json`.